packages/server-wallet/src/evm-validator.ts

```diff
@@ -16,8 +15,8 @@ const bytecodeCache: Record<string, string | undefined> = {};
  * Returns a promise that resolves to true if the validateTransition returns true false otherwise
  */
 export const validateTransitionWithEVM = async (
-  from: State,
-  to: State,
+  from: NitroState,
```

Comment:
Perhaps the PR's description could eg. explain the reasoning for this change? When I look through the commit history, I can't discern the reason for this change.

Reply:
The reason is that I found it more convenient to hoist the call to `toNitroState` up out of this function, because then I could also leverage `wireStateToNitroState` in the outer context if I wanted to, and get away with fewer `thisTypetoThatType` helpers.

 Also, I have an aspiration for this validation code to live in `nitro-protocol`, and this change would make that move easier to accomplish.